Jenkinsfile

```diff
@@ -797,7 +802,10 @@ pipeline {
                         }
                         */
                         always {
-                            archiveArtifacts artifacts: 'run_test.sh/**'
+				sh '''python utils/fix_cmocka_xml.py
+					ls -l test_results/'''
```

Comment:
Probably don't need this `ls -l ...` now that the questions of where/what files are being generated are have been resolved.

Reply:
removed ls -l